prism-core/src/main/java/io/avaje/prism/internal/APContextWriter.java

```diff
@@ -448,7 +458,6 @@ public static void write(PrintWriter out, String packageName, boolean moduleRead
             + "            .toString()\n"
             + "            .replaceFirst(id, \"\")\n"
             + "            .replaceFirst(\"/classes/java/main\", \"\")\n"
-            + "            .replaceFirst(\"/classes/java/test\", \"\")\n"
```

Comment:
Why was this removed? Is this a bug fix or just not necessary?

Reply:
oh woops